src/app/components/add-content/add-content.component.html

```diff
@@ -40,14 +88,16 @@ <h4 class="secondary-header text-right mt-2 mb-0">
                 </h4> 
                 <div class="custom-file mb-3">
                     <input type="file" class="custom-file-input" id="fileBrowser" (change)="onFileChange($event)"
-                            accept="application/pdf, .doc, .docx, .ppt, .pptx,">
-                    <label id="fileBrowserLabel" class="custom-file-label" for="fileBrowser">בחר קובץ...</label>
+                            accept="application/pdf, .doc, .docx, .ppt, .pptx,"/>
+                    <label #fileBrowserLabel id="fileBrowserLabel" class="custom-file-label" for="fileBrowser">בחירת קובץ...</label>
                 </div>
                 <div *ngIf="fileErrorSize" class="alert alert-danger" role="alert">
                     File size must be less then 10MB
                 </div>
             </div>
         </div>
+        <p class="text-danger text-center mt-3" *ngIf="!addContentForm.valid || !isFileSelected">אנא מלאו את כל הפרטים הנדרשים (מסומן ב-*).</p>

```

Comment:
Should `!addContentForm.valid || !isFileSelected` be put inside a function? It is used again a few lines below and may be confusing.